src/pages/community-stats/statistics.component.jsx

```diff
@@ -1,20 +1,18 @@
 import React from "react";
-import { Icon } from 'semantic-ui-react';
 import "./statistics.styles.scss";
 
 /**
- * @param {object} props 
+ * @param {object} props
  * @param {string} [props.className]
  * @param {string} props.title
  * @param {string} props.icon - Check this link to know the possible values: https://react.semantic-ui.com/elements/icon/
  * @param {any} props.value
  */
-const Statistics =({ className = "", title, icon, value }) => (
-  <div className={`statistic ${className}`}>
-    <span className="title">{title}</span>
-    <span className="value">{value}</span>
-    <Icon name={icon} size='huge' />
```

Comment:
is there a reason to remove this icon? If we are going to remove it we could also remove the `icon` prop since it's not used. But maybe this sort of change can come in another PR 😃

Reply:
@RemcoHalman if we are going to remove the icon, could you also delete the JSdoc of this component. When we make changes to the code we should try to make an effort to update the docs as well 👍 
Basically it's deleting the "@param {string} props.icon.....". Also, remove the style related to the icon since they are now unused CSS.

But before you do that, do you think the icon is unnecessary or it just doesn't fit with the design? I'm trying to understand the reason behind removing it before we make that decision 😃.